Experimental/src/org/sleuthkit/autopsy/experimental/autoingest/AutoIngestDashboard.java

```diff
@@ -46,8 +47,9 @@
 @SuppressWarnings("PMD.SingularField") // UI widgets cause lots of false positives
 final class AutoIngestDashboard extends JPanel implements Observer {
     
-    private final static String ADMIN_ACCESS_FILE_NAME = "adminAccess";
-    private final static String ADMIN_ACCESS_FILE_PATH = Places.getUserDirectory().getAbsolutePath() + File.separator + ADMIN_ACCESS_FILE_NAME;
+    private final static String ADMIN_ACCESS_FILE_NAME = "_aiaa"; // NON-NLS
+    private final static String ADMIN_ACCESS_FILE_PATH = Paths.get(Places.getUserDirectory().getAbsolutePath(),
+            "config" + File.separator + ADMIN_ACCESS_FILE_NAME).toString();
```

Comment:
Paths.get takes care of returning a Path with the path component separators between each item in the input list.  The concatenation with File.separator is not needed here.